src/main/zabbix/etc/zabbix/conf.d/sonicmq-monitoring.conf

```diff
@@ -0,0 +1,75 @@
+# Discovery
+UserParameter=sonicmq.broker.discovery,/etc/zabbix/scripts/sonicmq_discovery.sh Broker
+UserParameter=sonicmq.agent.discovery,/etc/zabbix/scripts/sonicmq_discovery.sh Agent
+UserParameter=sonicmq.agent_manager.discovery,/etc/zabbix/scripts/sonicmq_discovery.sh AgentManager
+UserParameter=sonicmq.connection.discovery,/etc/zabbix/scripts/sonicmq_discovery.sh Connection
+UserParameter=sonicmq.connection_aggregate.discovery,/etc/zabbix/scripts/sonicmq_discovery ConnectionAggregate
+UserParameter=sonicmq.queue.discovery,/etc/zabbix/scripts/sonicmq_discovery.sh Queue
+UserParameter=sonicmq.subscriber.discovery,/etc/zabbix/scripts/sonicmq_discovery.sh Subscriber
+
+# Broker items
```

Comment:
Please document here briefly what are the user parameter arguments (even though an advanced user can find them from script files).